Source/Core/Core/FifoPlayer/FifoRecordAnalyzer.cpp

```diff
@@ -227,78 +228,3 @@ void FifoRecordAnalyzer::WriteVertexArray(int arrayIndex, u8 *vertexData, int ve
 
 	FifoRecorder::GetInstance().WriteMemory(arrayStart, arraySize, MemoryUpdate::VERTEX_STREAM);
 }
-
-void FifoRecordAnalyzer::ProcessTexMaps()
```

Comment:
For future reference, the "above functions" (ProcessPreloadTexture and ProcessLoadTlut1) were in fact moved to VideoCommon in later commits.